src/main/java/de/bonndan/nivio/output/map/svg/HexPath.java

```diff
@@ -13,13 +14,19 @@
  */
 public class HexPath {
 
+    private final Item source;
+    private final Item target;
```

Comment:
This pushes domain objects down into the view level.

At least that looks like the wrong abstraction level for ``Item``s.

Reply:
Removed